Graphs/AbstractGraph.js

```diff
@@ -39,9 +39,9 @@ export default class AbstractGraph extends React.Component {
         this.setTooltip();
     }
 
-    componentWillReceiveProps(nextProps) {
-        this.setConfiguredProperties(nextProps, this.properties);
-        this.reInitializeTooltip(nextProps);
+    componentDidUpdate() {
```

Comment:
Here we cannot blindly update we need some sort of protection especially since the 2 functions below can be overwritten in the child class.